keras/engine/topology.py

```diff
@@ -1088,7 +1088,7 @@ def add_loss(self, losses, inputs=None):
         if hasattr(self, '_losses'):
             self._losses += losses
         # Update self._per_input_updates
-        if inputs == []:
+        if inputs is []:
```

Comment:
The above can be written as
```python 
if [] == inputs:
```
without triggering CNTK's operator `==`. Using `is` has its own issues.
```python
>>> a = [1]
>>> a.pop()
1
>>> a == []
True
>>> a is []
False
```